R/create_table_group_x_variable.R

```diff
@@ -0,0 +1,58 @@
+#' Create a wide table with indicators in the columns
+#'
+#' @param .results long table with the stats and analysis key
+#' @param analysis_key name of the columns of the analysis key, as character vector. Default is
+#' "analysis_key"
+#' @param value_columns names of the columns with the stats of interest, as character vector.
+#' Default is c("stat", "stat_low", "stat_upp")
+#'
+#' @return a wide table with grouping variables as rows and analysed variables as columns
+#' @export
+#'
+#' @examples
+#' create_table_group_x_variable(presentresults_resultstable, value_columns = "stat")
+#'
+create_table_group_x_variable <- function(.results,
+                                          analysis_key = "analysis_key",
+                                          value_columns = c("stat", "stat_low", "stat_upp")) {
+  verify_analysis_key(.results[[analysis_key]])
```

Comment:
Just curious to know if we need it or not? I believe this already exists in create_analysis_key_table().